Gopkg.toml

```diff
@@ -41,9 +44,9 @@
   name = "google.golang.org/grpc"
   version = "1.10.0"
 
-[[constraint]]
+[[override]]
   name = "github.com/golang/protobuf"
-  version = "1.0.0"
+  version = "=1.1.0"
```

Comment:
Is there a reason you version locked this and atlas-app-toolkit as exact matches? I'm not sure how override stanzas are propagated to projects that use this as a dependency, but it seems like that could cause problems.

Reply:
Yes. I agree with you. I fixed it.